sssd_test_framework/utils/override.py

```diff
@@ -0,0 +1,375 @@
+"Managing local users and groups."
+
+from __future__ import annotations
+
+import jc
+from pytest_mh import MultihostHost, MultihostUtility
+from pytest_mh.cli import CLIBuilder, CLIBuilderArgs
+from pytest_mh.ssh import SSHLog
+
+__all__ = [
+    "OverrideUtils",
+    "OverrideUser",
+    "OverrideGroup",
+]
+
+
+class OverrideUtils(MultihostUtility[MultihostHost]):
+    """
+    Management of override users and groups, sss_override.
+
+    .. note::
+
+        All changes are automatically reverted when a test is finished.
+    """
+
+    def __init__(self, host: MultihostHost) -> None:
+        """
+        :rtype: object
+        :param host: Remote host instance.
+        :type host: MultihostHost
+        """
+        super().__init__(host)
+
+        self.cli: CLIBuilder = CLIBuilder(host.ssh)
+        self._users: list[str] = []
+        self._groups: list[str] = []
+
+    def teardown(self) -> None:
+        """
+        Teardown
+        """
+        super().teardown()
```

Comment:
You need to remove added overrides here.

Reply:
I tried this, and it failed. sss_override command failed because the SSSD was no longer configured or running.  I'll check again to be sure.

Reply:
I see. I that case it may be good to make it part of the SSSDUtils to be able to teardown correctly.